File: peek_plugin_docdb/plugin-module/_private/tuples/StringCapToggleActionTuple.ts
                                       import {addTupleType, Tuple, TupleActionABC} from "@synerty/vortexjs";
import {docDbTuplePrefix} from "../PluginNames";

@addTupleType
export class StringCapToggleActionTuple extends TupleActionABC {
    static readonly tupleName = docDbTuplePrefix + "StringCapToggleActionTuple";

    documentId: number;

    constructor() {
        super(StringCapToggleActionTuple.tupleName)
    }
}

File: peek_plugin_docdb/_private/tuples/StringCapToggleActionTuple.py
from vortex.Tuple import addTupleType, TupleField
from vortex.TupleAction import TupleActionABC

from peek_plugin_docdb._private.PluginNames import docDbTuplePrefix


@addTupleType
class StringCapToggleActionTuple(TupleActionABC):
    __tupleType__ = docDbTuplePrefix + "StringCapToggleActionTuple"

    documentId = TupleField()
